### DISCUSSION TITLE
Email Subscription Opt In/Out for Collective/Host Admins

### DISCUSSION THREAD

**Discussion by SudharakaP** [opencollective/opencollective#4601] [Ideas]
Something I've been thinking over the weekend but thought I'll open a discussion first so to propose this idea and if it's something we decide is needed I'll create a project issue. 😄 

Currently collective or host admins are notified of all expenses, transactions etc. So for example, one can have ones email inbox filled with this kinda notifications. 

![image](https://user-images.githubusercontent.com/12435965/131408575-db9c4082-b68b-4457-af92-ec767167cd8d.png)

It might be worthwhile to have a opt out or opt in mechanism so that collective/host admins can select which type of notifications to receive and fine tune that in their profile settings. This way they could only opt in for notifications that they are interested in not every single notification. 😄 

cc: @opencollective/core-developers @alanna @BenJam

**Comment by alanna** [comment]
Yes please. I'll send you an export of all my filters which cover a lot of the notifications to make settings for.

**Comment by Betree** [comment]
There's already an "Unsubscribe" link at the bottom of emails, and we have https://github.com/opencollective/opencollective/issues/3357 designed to manage this from the interface.

**Reply by SudharakaP** [reply]
Thanks for linking the issue. I couldn't find this issue initially; that seems to cover the problem. 😃

**Reply by SudharakaP** [reply]
Although certain emails (such as request of virtual cards) do not have the unsubscribe button. Maybe we need to do analysis of different types of emails to see if all have unsubscribe button and make things consistent. 🤔
